### PR TITLE
Small additions/edits to ruler and alertmanager docs

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
Signed-off-by: Dimitar Dimitrov <dimitar.dimitrov@grafana.com>

#### What this PR does

I've been collecting these for some time, so committing them now.

- Fix help text of `mimirtool alertmanager load`
- Mentions a detail about alertmanager URL in mimirtool
- Cross-references mimirtool rules and the github action
- Adds a reference to Prometheus docs for alertmanager config

#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- NA Tests updated
- [x] Documentation added
- NA `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/operators-guide/tools/mimirtool.md [hunk 1]
```diff
@@ -72,6 +72,9 @@ The following sections outline the commands that you can run against Grafana Mim
 
 The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.
 
+> **Note:** The Mimir Alertmanager exposes the configuration API according to the path set by the `-server.path-prefix` flag. It does not use the path set by the `-http.alertmanager-http-prefix` flag.
+> With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion.
```
Comment (pracucci) [inline docs/sources/operators-guide/tools/mimirtool.md:76]: I would try to clarify it. I don't know if my edit is better, but I would try to make it easier to understand.

```suggestion
> With the default configuration of `-server.path-prefix`, `mimirtool` `--address` flag should be set to an URL with the `/` path.
```

File: docs/sources/operators-guide/tools/mimirtool.md [hunk 2]
```diff
@@ -131,6 +134,9 @@ The commands in this section enable you to perform the following actions:
 - Interact with individual rule groups in the Mimir ruler
 - Manipulate local rule files
 
+Some of the functionality that `mimirtool rules` offers is also available as a GitHub Action.
+For more information, refer to the [documentation of the action](https://github.com/grafana/mimir/blob/main/operations/mimir-rules-action/README.md).
```
Comment (pracucci) [inline docs/sources/operators-guide/tools/mimirtool.md:138]:
```suggestion
For more information, refer to the [documentation of Mimirtool Github Action](https://github.com/grafana/mimir/blob/main/operations/mimir-rules-action/README.md).
```

**Review by pracucci — APPROVED** [review]
Good job! I just left a couple of minor comments.

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 1 above)
Comment (dimitarvdimitrov) [reply docs/sources/operators-guide/tools/mimirtool.md:76]: I tried thinking of something more clear, but I couldn't do much better.

what do you think about just giving an example?
```suggestion
> With the default configuration of `-server.path-prefix`, the Alertmanager URL used as the `mimirtool` `--address` flag has no path portion: for example `http://alertmanager.mimir.example.com/`.
```

Comment (pracucci) [reply docs/sources/operators-guide/tools/mimirtool.md:76]: The example could help. @osg-grafana could help further :)

File: docs/sources/operators-guide/architecture/components/alertmanager.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Each tenant has an isolated alert routing configuration and Alertmanager UI.
 ### Tenant configurations
 
 Each tenant has an Alertmanager configuration that defines notifications receivers and alerting routes.
-The Mimir Alertmanager uses the same configuration file used by the Prometheus Alertmanager.
+The Mimir Alertmanager uses the same [configuration file used by the Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/configuration/#configuration-file).
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/alertmanager.md:22]:
```suggestion
The Mimir Alertmanager uses the same [configuration file](https://prometheus.io/docs/alerting/latest/configuration/#configuration-file) that the Prometheus Alertmanager uses.
```

File: docs/sources/operators-guide/architecture/components/ruler/index.md [hunk 4]
```diff
@@ -101,7 +101,19 @@ Ruler replicas form their own [hash ring]({{< relref "../../hash-ring/index.md"
 
 To configure the rulers' hash ring, refer to [configuring hash rings]({{< relref "../../../configure/configuring-hash-rings.md" >}}).
 
-## HTTP configuration API
+## Managing alerting and recording rules
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/ruler/index.md:104]:
```suggestion
## Manage alerting and recording rules
```

File: docs/sources/operators-guide/architecture/components/ruler/index.md [hunk 5]
```diff
@@ -101,7 +101,19 @@ Ruler replicas form their own [hash ring]({{< relref "../../hash-ring/index.md"
...
+
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/ruler/index.md:105]:
```suggestion

There is more than one way to manage alerting and recording rules.

```

File: docs/sources/operators-guide/architecture/components/ruler/index.md [hunk 6]
```diff
@@ -101,7 +101,19 @@ Ruler replicas form their own [hash ring]({{< relref "../../hash-ring/index.md"
...
+### Via the `mimirtool` CLI tool
+
+The `mimirtool rules` command offers utility subcommands for linting, formatting and uploading rules to Grafana Mimir.
+Refer to the [documentation of `mimirtool rules`]({{< relref "../../../tools/mimirtool.md#rules" >}}) for more information.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/ruler/index.md:109]:
```suggestion
The `mimirtool rules` command offers utility subcommands for linting, formatting, and uploading rules to Grafana Mimir.
For more information, refer to the [`mimirtool rules`]({{< relref "../../../tools/mimirtool.md#rules" >}}).
```

File: docs/sources/operators-guide/architecture/components/ruler/index.md [hunk 7]
```diff
@@ -101,7 +101,19 @@ Ruler replicas form their own [hash ring]({{< relref "../../hash-ring/index.md"
...
+
+### Via the `grafana/mimir/operations/mimir-rules-action` GitHub Action
+
+`mimir-rules-action` wraps some of the functionality of `mimirtool rules` in a GitHub action.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/architecture/components/ruler/index.md:113]:
```suggestion
The GitHub Action `mimir-rules-action` wraps some of the functionality of `mimirtool rules`.
```
The preference here is to not start a sentence with a lowercase.

File: docs/sources/operators-guide/tools/mimirtool.md [hunk 8]
```diff
@@ -72,6 +72,9 @@ The following sections outline the commands that you can run against Grafana Mim
 
 The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.
 
+> **Note:** The Mimir Alertmanager exposes the configuration API according to the path set by the `-server.path-prefix` flag. It does not use the path set by the `-http.alertmanager-http-prefix` flag.
```
Comment (osg-grafana) [inline docs/sources/operators-guide/tools/mimirtool.md:75]:
```suggestion
> **Note:** The Mimir Alertmanager exposes the configuration API based on the path that you set via the `-server.path-prefix` flag; it does not use the path of the `-http.alertmanager-http-prefix` flag.
```
I am not sure if the user has set both of these paths or one of them. Therefore, I changed this to possession (the path of the ...).

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 1 above)
Comment (osg-grafana) [inline docs/sources/operators-guide/tools/mimirtool.md:76]:
```suggestion
> If you run Mimir with the default configuration (`-server.path-prefix`, where the default value is `/`), then only set the hostname for the `mimirtool --address` flag; do not set a path-specific address. For example, `/` is correct, and `/alertmanager` is incorrect.
```

File: pkg/mimirtool/commands/alerts.go [hunk 9]
```diff
@@ -74,8 +74,8 @@ func (a *AlertmanagerCommand) Register(app *kingpin.Application, envVars EnvVarN
 
 	alertCmd.Command("delete", "Delete the Alertmanager configuration that is currently in the Grafana Mimir Alertmanager.").Action(a.deleteConfig)
 
-	loadalertCmd := alertCmd.Command("load", "Load a set of rules to a designated Grafana Mimir endpoint").Action(a.loadConfig)
-	loadalertCmd.Arg("config", "alertmanager configuration to load").Required().StringVar(&a.AlertmanagerConfigFile)
+	loadalertCmd := alertCmd.Command("load", "Load Alertmanager tenant configuration and template files to Grafana Mimir.").Action(a.loadConfig)
```
Comment (osg-grafana) [inline pkg/mimirtool/commands/alerts.go:77]:
```suggestion
	loadalertCmd := alertCmd.Command("load", "Load Alertmanager tenant configuration and template files into Grafana Mimir.").Action(a.loadConfig)
```

**Review by osg-grafana — APPROVED** [review]
Unblocking with relatively important clarification about one bit

File: pkg/mimirtool/commands/alerts.go (hunk 9 above)
Comment (osg-grafana) [reply pkg/mimirtool/commands/alerts.go:77]: (Double checking this preposition with fellow writers, and can catch this in another PR if you want to leave it or change it.)

File: docs/sources/operators-guide/tools/mimirtool.md (hunk 1 above)
Comment (dimitarvdimitrov) [reply docs/sources/operators-guide/tools/mimirtool.md:76]: we've used "`mimirtool` `--address` flag" in another article (alertmanager.md), should I update it there too?

File: docs/sources/operators-guide/tools/mimirtool.md [hunk 10]
```diff
@@ -131,6 +134,9 @@ The commands in this section enable you to perform the following actions:
 - Interact with individual rule groups in the Mimir ruler
 - Manipulate local rule files
 
+Some of the functionality that `mimirtool rules` offers is also available as a GitHub Action.
+For more information, refer to the [documentation of Mimirtool Github Action](https://github.com/grafana/mimir/blob/main/operations/mimir-rules-action/README.md).
```
[the remark this answers is not in the document]
Comment (dimitarvdimitrov) [reply docs/sources/operators-guide/tools/mimirtool.md:138]: it cant be relref since we don't publish that readme anywhere

**Pull request merged**